### DISCUSSION TITLE
My Solutions tab displays submissions incorrectly

### DISCUSSION THREAD

**Discussion by aarthvaen** [cssbattle/forum#300] [Bugs reports]
When trying to load a solution, all the code, as well as the **Load Code** button, appears to go outside of the **My Solutions** tab.
Force-refreshing the cache doesn't seem to resolve it.

![c5TpjLsGYH](https://user-images.githubusercontent.com/44089986/186661246-2962d551-7030-42bf-bf4f-9eb4213183fd.gif)

**Comment by chinchang** [comment]
fixed.
